fooof/utils/data.py

```diff
@@ -101,23 +104,35 @@ def interpolate_spectrum(freqs, powers, interp_range, buffer=3):
     >>> freqs, powers = interpolate_spectrum(freqs, powers, [58, 62])
     """
 
-    # Get the set of frequency values that need to be interpolated
-    interp_mask = np.logical_and(freqs >= interp_range[0], freqs <= interp_range[1])
-    interp_freqs = freqs[interp_mask]
+    # If given a list of interpolation zones, recurse to apply each one
```

Comment:
After working on the #198, do you think it would be useful to generalize `interpolate_spectrum` to `interpolate_spectra`. I could see a case where one would want to remove line noise systematically across multiple spectra. Looping this function would be easy enough, however.

Reply:
This is a great idea! Just like the other util function, `trim_spectrum`, does, `interpolate_spectrum` (or renamed to `spectra` should totally support 2D arrays. If there is line noise in a dataset, it should be consistent across channels, and so applying this to a 2D collection of PSDs is if anything likely to be more common than to 1D. 

Do you want to try out generalizing the interpolation to 2D? It can go into a new PR I think.